requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 coverage>=4.4.0
 matplotlib>=2.1
 black==21.4b2
-pylint>=2.7.1
+pylint>=2.8.3
```

Comment:
We probably should pin this (and astroid) to an exact version to avoid spurious failures caused by new releases

Reply:
Thanks. The version is pinned by `constraint.txt`. I forgot to upgrade...